LibreHardwareMonitor/UI/MainForm.Designer.cs

```diff
@@ -1,7 +1,7 @@
 // This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
 // If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
 // Copyright (C) LibreHardwareMonitor and Contributors.
-// Partial Copyright (C) Michael M�ller <mmoeller@openhardwaremonitor.org> and Contributors.
+// Partial Copyright (C) Michael Möller <mmoeller@openhardwaremonitor.org> and Contributors.
```

Comment:
Undo + all space changes in this file.

Reply:
I think this is due to a codepage change, do you know what format (UTF8 or something else) this project is in?

Reply:
Default VS format I guess, no idea :)

Reply:
this is 100% a codepage problem, and I can't resolve it properly without setting the project's codepage as this is an autogenerated file - any ideas what it should be?

Reply:
I think it's just ANSI.